keyboards/polykybd/.claude/skills/add-polykybd-shortcut-hint/hint_preview.py

```
@@ -0,0 +1,118 @@
+per font. Hints sit RIGHT-of-center (~x46-50 in the 72px window), not mathematically
+centered — match that. Wide emoji need FEWER leading spaces than narrow symbol glyphs.
+"""
+import argparse, sys, os
+
+# Resolve the gfx_font loader from the PolyKybdHost tools dir.
+HERE = os.path.dirname(os.path.abspath(__file__))
+for cand in ("tools",  # cwd == PolyKybdHost
+             os.path.join(HERE, "../../../../../PolyKybdHost/tools"),
+             "/home/user/PolyKybdHost/tools"):
+    if os.path.exists(os.path.join(cand, "gfx_font.py")):
+        sys.path.insert(0, cand); break
```

Ruff (0.15.18)

[error] Multiple statements on one line (semicolon)

(E702)